src/app/layout.tsx

```diff
@@ -12,6 +12,9 @@ const inter = Inter({ subsets: ["latin"] });
 export const metadata: Metadata = {
   title: "Hive",
   description: "A fresh start",
+  icons: {
+    icon: "http://website.sphinx.chat/wp-content/uploads/2021/10/Favicon_64x64.jpg",
```

Comment:
it it from workflow? we should use this link

Reply:
this is just for testing, i wanted to create the PR
we shouldn't merge this one as is since it's the wrong image